Nested stores event bubbling

samuelsimoes opened the pull request:
#21

sobrinho commented on src/fluxo.object_store.js:
```diff
@@ -117,6 +151,10 @@ export default class {
+    if (this.data[attribute] instanceof this.constructor) {
+      this.stopListenStoreAttribute(attribute);
+    }
+
```
`this.constructor` will be always `Fluxo.ObjectStore`?

If I have a store A inheriting from `Fluxo.ObjectStore` and then a store B inheriting from A, the constructor of B will be `Fluxo.ObjectStore` or `A`?

samuelsimoes replied:
It's wrong... Must be a comparison with `Fluxo.ObjectStore`, I will fix.

samuelsimoes commented:
@sobrinho fixed :point_up:

sobrinho commented:
@samuelsimoes the same applies to the others `this.constructor`, no?

samuelsimoes commented:
If I understood your question the B instance will respond true for ObjectStore, A and B, like this:
![fluxo](https://cloud.githubusercontent.com/assets/1456228/13205131/3ddbd172-d8bf-11e5-9f2f-9f70d5da5373.png)

samuelsimoes committed:
Nested stores event bubbling

Now, if you set an attribute with a fluxo object store (this includes
the collection) all children events will be propagated to parents.
